src/adapter.js

```diff
@@ -173,12 +173,11 @@ var createMochaReporterConstructor = function (tc, pathname) {
   }
 }
 /* eslint-disable no-unused-vars */
-var createMochaStartFn = function (mocha) {
+var createMochaStartFn = function (karma, mocha) {
```

Comment:
Make karma the second argument so existing code calling with one argument will work.